src/iop/filmicrgb.c

```diff
@@ -1611,13 +2735,81 @@ void gui_init(dt_iop_module_t *self)
                                                         "this should be 100%\nexcept if you want a faded look"));
   g_signal_connect(G_OBJECT(g->white_point_target), "value-changed", G_CALLBACK(white_point_target_callback), self);
 
-  // power/gamma slider
-  g->output_power = dt_bauhaus_slider_new_with_range(self, 1.0, 10.0, 0.1, p->output_power, 2);
-  dt_bauhaus_widget_set_label(g->output_power, NULL, _("target power transfer function"));
-  gtk_box_pack_start(GTK_BOX(page3), g->output_power, FALSE, FALSE, 0);
-  gtk_widget_set_tooltip_text(g->output_power, _("power or gamma of the transfer function\nof the display or color space.\n"
-                                                 "you should never touch that unless you know what you are doing."));
-  g_signal_connect(G_OBJECT(g->output_power), "value-changed", G_CALLBACK(output_power_callback), self);
+
+  // Color science
+  g->version = dt_bauhaus_combobox_new(self);
+  dt_bauhaus_widget_set_label(g->version, NULL, _("color science"));
+  dt_bauhaus_combobox_add(g->version, _("v3 (2019)"));
+  dt_bauhaus_combobox_add(g->version, _("v4 (2020)"));
+  gtk_widget_set_tooltip_text(g->version, _("v3 is darktable 3.0 desaturation method, same as color balance.\n"
+                                            "v4 is a newer desaturation method, based on spectral purity of light."));
```

Comment:
"spectral purity of light" - marketing buzzword? :-) I'm not expert in photo processing, so could be that it is a well-known term in this field?

Reply:
No marketing. A white spectrum is a spectrum where every frequency has an equal weight. A pure spectrum is… well not a spectrum anymore, but almost a single frequency (kind of a laser mono-wavelength). So the idea of the v4 is to sharpen or even the shape of the light spectrum, making it more pure or more white.

Reply:
Thanks for your explanation. Already thought that you are not using the term for marketing :-) What market - DT is free!